packages/lesswrong/components/common/MixedTypeFeed.tsx

```diff
@@ -40,6 +40,7 @@ const getQuery = ({resolverName, resolverArgs, fragmentArgs, sortKeyType, render
         __typename
         cutoff
         endOffset
+        sessionId
```

Comment:
I think this is the wrong place for this field; there are other usages of `MixedTypeFeed` where there is no `sessionId`, eg the tag history page.